config/install/migrate_plus.migration.y_lb_demo_block_partners.yml

```diff
@@ -69,18 +60,18 @@ process:
   field_section_subtitle: subtitle
   field_partners:
     plugin: sub_process
-    source: content_ids
+    source: tier_ids
     process:
       target_id:
         plugin: migration_lookup
         migration:
-          - y_lb_demo_block_partners_item
-        source: content_id
+          - y_lb_demo_block_partners_tier
+        source: tier_id
```

Comment:
can we understand what is the difference between removed `y_lb_demo_block_partners_item` and created `y_lb_demo_block_partners_tier` ? 
They seems like similar, but new one does work, old one - does not